dbt_platform_helper/commands/pipeline.py

```diff
@@ -62,161 +50,11 @@ def generate(terraform_platform_modules_version, deploy_branch):
     This command does the following in relation to the codebase pipelines:
     - Generates the copilot pipeline manifest.yml for copilot/pipelines/<codebase_pipeline_name>
     """
```

Comment:
All the following code has moved into the domain/pipelines.py file and the Pipelines class.